docs/usage-as-end-user.md

```diff
@@ -162,3 +162,16 @@ Every GCP shoot cluster that has at least Kubernetes v1.18 will be deployed with
 It is compatible with the legacy in-tree volume provisioner that was deprecated by the Kubernetes community and will be removed in future versions of Kubernetes.
 End-users might want to update their custom `StorageClass`es to the new `pd.csi.storage.gke.io` provisioner.
 Shoot clusters with Kubernetes v1.17 or less will use the in-tree `kubernetes.io/gce-pd` volume provisioner in the kube-controller-manager and the kubelet.
+
+## WorkerConfig
+
+The worker config configuration mainly contains Local SSD interface for the GCP,
+If you attach the disk with `SCRATCH` type, either an NVMe interface or a SCSI interface must be specified.
+An example `WokerConfig` for the GCP looks as follows:
+
+```$xslt
+apiVersion: gcp.provider.extensions.gardener.cloud/v1alpha1
+kind: WokerConfig
+volume:
```

Comment:
Actually, only the disk type in DataVolumes is `SCRATCH` type, the `NVME` interface will be used.

I think it will be complicated to allow configuring the `interface` per data volume because the disk type of data volume maybe `pd-ssd`, for `pd-ssd`, the interface is not needed.

Reply:
Sure, it's okay, but then we should have proper validation in place to protect users from misconfiguring it.